src/content/docs/new-relic-solutions/build-nr-ui/build-ab-app/add-ui.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Add user interface components to your application'
+metaDescription: 'Add user interface components to your application'
+freshnessValidatedDate: never
+---
+
+<Callout variant="course">
+
+This lesson is part of a course that teaches you how to build a New Relic application from the ground up. If you haven't already, check out the Overview.
+
+Each lesson in the course builds upon the last, so make sure you've completed the last lesson, Add a chart group, before starting this one.
+
+</Callout>
+
+If an application is organized, it is more readable and more understandable. In your A/B test application, you have a lot of charts, but no organization. In the next few lessons, you'll use user interface components to bring some organization to your application and provide new functionality. First, you'll use a `Grid` component to arrange your charts to match the layout in your design guide. Second, you'll add headings to clarify what each chart represents. Third, you'll add descriptions for your A and B design versions to the top of your app. Finally, you'll create a section for ending your A/B test.
+
+In the next lesson, you arrange your charts to look like they do in your design guide.
+
+<Callout variant="course">
```

Comment:
Can we leave this callout as `course`? I've replaced it with `tip` in the files I've done. Can I keep `course` from now on even if we don't have this kind of callout in our guide style?